modules/ROOT/pages/_partials/task-configure-proxy.adoc

```diff
@@ -127,6 +127,7 @@ You should now see the **TLS Context for Inbound Traffic** field enabled.
 . Click *Advanced Options*.
 . In *Proxy Version*, select *latest*. +
 This value ensures that your API proxy uses the latest released proxy version.
+If you update proxy version in exchange, return to this step to reconfigure your proxy to the latest version.
```

Comment:
I added this because the ticket is originally in reference to the configure page, but was really just providing information about normal API's. I don't think it hurts to have it here.

Reply:
I believe that they were probably mixing concepts in the ticket. Users can't upload new proxy versions to exchange, we upload them internally and they just see new proxy versions available. Maybe we can rephrase this to something like:

```
If a new proxy version is available in Anypoint platform, you can return to this step to reconfigure your proxy to the latest version.
```